### ISSUE TITLE
Improve version dropdowns

### ISSUE THREAD

**Issue by MartinKolarik** [jsdelivr/www.jsdelivr.com#601]
The version dropdowns, as they are now, are quite messy for projects that publish prereleases, e.g.:
![image](https://github.com/jsdelivr/www.jsdelivr.com/assets/6192491/e8da8cde-66fe-4baa-8e1f-98c61687051e)

How about we split the version list into three sections with a separator in between:
1. All stable, non-deprecated versions.
2. All prereleases.
3. Any deprecated versions.

Each section would be sorted on its own from the most recent.

**Comment by MartinKolarik** [comment]
We don't have the deprecation info yet, but let's try this out in a simpler version first: 

1. All stable versions.
2. All prereleases.

**Comment by mpourismaiel** [comment]
Is something like this what you are looking for?

```
┌──────────────────┐
│ Static           │
│ Latest Patch     │
│ Latest Minor     │
│ Latest Major     │
├──────────────────┤
│ ▼ Version List   │
│                  │
│  Stable Releases │
│  • 2.3.4         │
│  • 2.3.3         │
│  • 2.2.0         │
│  • 1.9.8         │
│                  │
├──────────────────┤
│  Pre-releases    │
│  • 2.4.0-beta.2  │
│  • 2.4.0-beta.1  │
│  • 2.3.5-rc.1    │
├──────────────────┤
```

One section after the other, all in a single scrollable area

**Comment by MartinKolarik** [comment]
Yes, except no `Version List` header, but `Stable Release` and `Pre-releases` headers maybe could be there.

**Comment by MartinKolarik** [comment]
Note that we've also added filtering since the issue was created (see https://www.jsdelivr.com/package/npm/react). When showing filtered results, the list would also be split into stable and pre-release sections.
